cypress/integration/sidebar.spec.js

```diff
@@ -0,0 +1,22 @@
+const autoRecord = require('cypress-autorecord');
+
+describe('The Sidebar', () => {
+  autoRecord();
+
+  beforeEach(() => {
+    cy.login()
+  })
+
+  it('click in the user avatar opens sidebar', function () {
```

Comment:
@rafayet-monon The spec should describe about `sidebar` not about login and logout? 🤔
In my idea, it should be something like the sidebar open and close.
What do you think?
If we want to check about `login` `logout`, it should be in the another file

Reply:
I agree, @Lahphim. Fixed in [fd49f0a](https://github.com/rafayet-monon/surveyor/pull/25/commits/fd49f0a34f10bd3490524fed5d51d1614938c7ff)